xbmc/FileItem.cpp

```diff
@@ -2021,7 +2021,7 @@ void CFileItemList::Sort(SortDescription sortDescription)
   }
 
   // replace the current list with the re-ordered one
-  m_items.assign(sortedFileItems.begin(), sortedFileItems.end());
+  m_items = std::move(sortedFileItems);
```

Comment:
No, that's completely wrong! You didn't read the code. Or maybe you just don't know C++ well enough to understand the code.
sortedFileItems is a std::vector. You can move a vector even if the items are not movable. Actually, it doesn't matter if a vector's items are movable.